desc/examples/WISTELL-A

```diff
@@ -8,44 +7,21 @@ NFP = 4
 Psi = 6.14553547E-01
 
 # spectral resolution
-L_rad  =  6:6:24
-M_pol  =  6, 12
-N_tor  =  0x2, 6x3
-M_grid = 12, 16
-N_grid =  0x2, 9x3
-
-# continuation parameters
-bdry_ratio = 0, 0, 0.33, 0.66, 1
-pres_ratio = 0
-pert_order = 2
+M_pol  =  12
+N_tor  =  6
+M_grid = 16
+N_grid =  9
 
 # solver tolerances
 ftol = 1e-2
 xtol = 1e-6
 gtol = 1e-6
-maxiter = 150
+maxiter = 100
 
 # solver methods
 optimizer         = lsq-exact
-objective         = force
-spectral_indexing = fringe
-
-# pressure and rotational transform profiles
-l:   0	p =   0.00000000E+00	i =   1.02962652e+00
-l:   2	p =   0.00000000E+00	i =  -1.53836061e-03
-l:   4	p =   0.00000000E+00	i =  -1.66978008e-01
-l:   6	p =   0.00000000E+00	i =   5.88058698e-01
-l:   8	p =   0.00000000E+00	i =  -5.52459520e-01
-l:  10	p =   0.00000000E+00	i =   2.02468032e-01
-
-# magnetic axis initial guess
-n:   0	R0 =   2.01244000E+00	Z0 =   0.00000000E+00
-n:   1	R0 =   4.05567000E-01	Z0 =   0.00000000E+00
-n:   2	R0 =   2.31526000E-02	Z0 =   0.00000000E+00
-n:   3	R0 =   4.72049000E-03	Z0 =   0.00000000E+00
-n:  -1	R0 =   0.00000000E+00	Z0 =  -2.96193000E-01
-n:  -2	R0 =   0.00000000E+00	Z0 =  -2.18210000E-02
-n:  -3	R0 =   0.00000000E+00	Z0 =  -2.54949000E-03
+objective         = vacuum
+spectral_indexing = ansi
```

Comment:
we dont include p or i or c bc it defaults to p and c of 0

Reply:
yeah vacuum doesn't need any profiles